### PR TITLE
Update create facility steps design only

### LINKED ISSUE #9306: Design Update: DeviceName
# Overview

- The Device name page has a new design.

# Acceptance Criteria

- [ ] Apply the new designs in Figma ([Desktop](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=1%3A527) | [Mobile](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=0%3A1))

### LINKED ISSUE #9307: New Component & Onboarding Step: What kind of device is this?
# Overview

- The "What kind of device is this" step is new in this flow, a new component will need to be made

# Acceptance Criteria

- [ ] Apply the new designs in Figma ([Desktop](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=1088%3A14095) | [Mobile](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=1182%3A14379))

### LINKED ISSUE #9308: Design Update: Create new or import facility
# Overview

- The page where user selects to create new facility or import has a new design. The component has been renamed from `SetupMethod` to `FullDeviceNewOrImportFacility` - which, while more verbose is also more descriptive of its context and purpose :) 

# Acceptance Criteria

- [ ] Apply the new designs in Figma ([Desktop](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=1%3A535) | [Mobile](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=1%3A1494))
- [ ] Ensure the component sends the necessary event `value` property ("new" or "import") when it moves the state machine along

### LINKED ISSUE #9309: Design Update: All steps in CreateNewFacility component
# Overview

- The whole CreateNewFacility flow has a new design.

# Acceptance Criteria

- [ ] Apply the new designs in Figma
- [ ] Update the CreateFacilitySetup component as needed

## Desktop

Enable Guest Access (Figma - [Desktop](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=1%3A820) | [Mobile](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=1%3A1689))
Allow learners to join this facility? (Figma - [Desktop](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=1%3A832) | [Mobile](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=1%3A1698))
Enable password on learner accounts? (Figma - [Desktop](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=1%3A843) | [Mobile](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=1%3A1706))
Responsibilities as an admin (Figma - [Desktop](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=1%3A903) | [Mobile](https://www.figma.com/file/MpCG7r5PULOCJsEvnTSti8/Android-Designs-2022?node-id=1%3A1828))

### PR DESCRIPTION
**Pull request by marcellamaki**
## Summary

Creates "Set Up Learning Facility" form
Updates the design for the 5-sub step "Create Facility" new form

## References

Fixes #9309
Fixes #9307
Fixes #9308
Fixes #9306

Screen Shot 2022-05-31 at 3 03 28 PM

https://user-images.githubusercontent.com/17235236/171421854-d0e7e90c-d09e-4d54-aa24-21377e29a262.mp4

## Reviewer guidance
This is mostly code review for design compliance.  Due to the routes here that are not part of the state machine, I suggest that we update the state machine to include these steps as well, even if they follow the same route pattern, i.e. `create_facility/:step_id`

In this current version, the button _does_ work to update the screens between the create facility steps, but data is not validated or saved. I think updating these to be part of the state machine will also help with using the new version of the OnboardingStepBase better, and allow a better flow when advancing between the steps with validations and committing to the store. Ideally, in a follow up issue that includes the state machine, some of the vue parts of this code will be updated to, to take full advantage of having one base component, and reducing/eliminating work-around and redundancies that are currently present. It will follow the same pattern of the OnboardingStepBase, rather than have it be a nested component that then doesn't have the full props options passed to it.

----

## Testing checklist

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

## PR process

- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/RequirePasswordForLearnersForm.vue [hunk 1]
```diff
@@ -1,60 +1,80 @@
 <template>
 
-  <YesNoForm
```
Comment (nucleogenesis) [inline kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/RequirePasswordForLearnersForm.vue:3]: I think this component is no longer in use with these changes so we can delete it.

**Review by nucleogenesis — APPROVED** [review]
Overall looks good. Can remove the YesNo component - but not a blocker as we can merge and remove it in our continuing work later today.

File: kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/RequirePasswordForLearnersForm.vue (hunk 1 above)
Comment (marcellamaki) [reply kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/RequirePasswordForLearnersForm.vue:3]: Ah, good point!

**Comment by marcellamaki** [comment]
@nucleogenesis -- I have actually made some changes to this rather than just having _strictly_ the design elements as I was anticipating, because just doing the design as I originally did cause quite a lot of duplicate work and problems. 

So, I have updated this PR to update also the wizard machine and routes. Now, the full flow should be working. I will update the tests to be more suitably written to the new pattern (right now, they are all testing the "create facility components" for the submit event, but it really lives in the step component). Would appreciate your re-review and thoughts!

https://user-images.githubusercontent.com/17235236/172718869-3b61e781-9453-4207-b561-25e046521d34.mp4

File: kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SettingUpKolibri.vue [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<template>
+
+  <p>~INTERSTITIAL~ </p>
```
Comment (nucleogenesis) [inline kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SettingUpKolibri.vue:3]: I think that [this page](https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/setup_wizard/assets/src/views/submission-states/LoadingPage.vue) is the one currently used for this

File: kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SetUpLearningFacilityForm.vue [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    inject: ['wizardService'],
+    data() {
+      return {
+        Options,
+        selected: Options.NEW,
+      };
+    },
+    methods: {
+      handleContinue() {
+        console.log('handle continue:', this.selected);
```
Comment (nucleogenesis) [inline kolibri/plugins/setup_wizard/assets/src/views/onboarding-forms/SetUpLearningFacilityForm.vue:48]: Errant console.log

**Review by nucleogenesis — APPROVED** [review]
Just a minor non-blocking note and a sneaky console.log - otherwise everything looks great!

**Review by nucleogenesis — COMMENTED** [review]
Another thing to note is that some of the radio button groups don't have a default selected answer and are all blank until you select one but [KDS requires one to always be selected](https://design-system.learningequality.org/kradiobutton/#guidelines).

Not blocking as this is a design-only PR and that can be hashed out as we continue wiring everything up.

**Pull request merged**